### PR TITLE
Use new api in Qt app (control in Node)

### PR DESCRIPTION
**Pull request by luminosuslight**
Problems left:
* feed is not displayed

### CODE REVIEW

File: source/rss-qt-node/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    # Copy node.dll
+    add_custom_command(TARGET ${target} POST_BUILD # Adds a post-build event to MyTest
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different  # which executes "cmake - E copy_if_different..."
+        ${CMAKE_SOURCE_DIR}/externals/node-v9.0.0/Release/node.dll      # <--this is in-file
+        $<TARGET_FILE_DIR:${target}>)                 # <--this is out-file path
+
+    # Copy Qt dependencies
+    find_program(WINDEPLOYQT_EXECUTABLE NAMES windeployqt HINTS ${Qt5_DIR}/../../../ ENV QTDIR PATH_SUFFIXES bin)
+    add_custom_command(TARGET ${target} POST_BUILD
+    COMMAND ${WINDEPLOYQT_EXECUTABLE} --no-patchqt --qmldir ${CMAKE_SOURCE_DIR}/source/qt-app $<TARGET_FILE:${target}>)
```
Comment (cmfcmf) [inline source/rss-qt-node/CMakeLists.txt]: wrong path

File: source/rss-qt-node/rss_feed.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      this.emit('error', new Error('Bad status code'));
+    }
+    else {
+      cppDemoModule.clearFeed();
+      stream.pipe(feedparser);
+    }
+  });
+ 
+  feedparser.on('error', function (error) {
+    // always handle errors
```
Comment (cmfcmf) [inline source/rss-qt-node/rss_feed.js]: The comment is a bit misleading, isn't it?

File: source/CMakeLists.txt [hunk 3]
```diff
@@ -17,6 +17,7 @@ set(IDE_FOLDER "")
 # Applications
 set(IDE_FOLDER "Applications")
 add_subdirectory(qt-app)
+add_subdirectory(rss-qt-node)
```
Comment (cmfcmf) [inline source/CMakeLists.txt]: The naming is a bit confusing. Maybe rename the three to 
- node-qt-rss
- node-lib-qt-rss
- node-lib-cli

File: source/rss-qt-node/rss_feed.js (hunk 2 above)
Comment (justus-hildebrand) [reply source/rss-qt-node/rss_feed.js]: yeah. should probably be something like "catch all parser errors but don't handle them"

File: source/CMakeLists.txt (hunk 3 above)
Comment (justus-hildebrand) [reply source/CMakeLists.txt]: :+1:

File: source/rss-qt-node/CMakeLists.txt (hunk 1 above)
Comment (justus-hildebrand) [reply source/rss-qt-node/CMakeLists.txt]: thanks for pointing this out, must've missed these paths when I changed the directory structure.
fixed in 77d6b2e.

File: source/rss-qt-node/rss_feed.js (hunk 2 above)
Comment (justus-hildebrand) [reply source/rss-qt-node/rss_feed.js]: fixed in 77d6b2e.

File: source/CMakeLists.txt (hunk 3 above)
Comment (justus-hildebrand) [reply source/CMakeLists.txt]: fixed in 254e759 and 77d6b2e

**Comment by cmfcmf** [comment]
Merging this, because the remaining problem needs to be fixed in the node repo.

**Pull request merged**